client/state/inline-help/reducer.js

```diff
@@ -70,6 +72,16 @@ export const search = withoutPersistence(
 		action
 	) => {
 		switch ( action.type ) {
+			case INLINE_HELP_SEARCH_RESET:
+				return {
+					searchQuery: '',
+					items: {
+						...state.items,
+						'': action.searchResults,
```

Comment:
:interesting:

Reply:
Better ideas welcome 👍 

This is how the state is stored. 

![Screen Shot 2020-06-18 at 15 00 16](https://user-images.githubusercontent.com/444434/85031092-2a2beb00-b176-11ea-809a-c217e1ef7f36.png)